docs/development/security/index.asciidoc

```diff
@@ -1,12 +1,22 @@
 [[development-security]]
 == Security
 
+=== Overview
+
+Feature development must consider user access permissions. Its easy for a design to assume access to several APIs which may have independent availability. Feature implementation must account for this although clever use of apis may sidestep apparent restrictions.
```

Comment:
Would you mind elaborating upon:
> although clever use of apis may sidestep apparent restrictions.

I'm not sure that I'm following.

Reply:
It might be best to just remove this. Might have been too particular to my situation.